client/state/partner-portal/stored-cards/actions.ts

```diff
@@ -79,6 +79,11 @@ export const deleteStoredCard =
 					type: 'STORED_CARDS_UPDATE_IS_PRIMARY_COMPLETED',
 					payment_method_id: response.primary_payment_method_id,
 				} );
+
+				dispatch( {
+					type: JETPACK_PARTNER_PORTAL_PARTNER_SET_HAS_VALID_PAYMENT_METHOD,
+					hasValidPaymentMethod: !! response.primary_payment_method_id,
+				} );
```

Comment:
Shouldn't we use the new setPartnerHasValidPaymentMethod() action here? This might end up being irrelevant based on my other comment regarding the actions used.

Reply:
Typing wouldn't let me use PartnerPortalThunkActions here, but I will consider your suggestion about using the same actions that are already being fired to add/remove cards to update the flag, it will suit well for our current need.